Cargo.toml

```diff
@@ -4,7 +4,10 @@ members = ["cli", "lib"]
 resolver = "2"
 
 [workspace.package]
-rust-version = "1.65"
+rust-version = "1.71"
```

Comment:
This shouldn't be raised without strong reasons, it's a minimum supported rust version.

Reply:
And it may be different for different crates and maximally lowest for the lib binding crate, but no one requested that or complained on it yet.